superset/config.py

```diff
@@ -77,7 +77,7 @@
 WTF_CSRF_EXEMPT_LIST = []
 
 # Whether to run the web server in debug mode or not
-DEBUG = False
+DEBUG = True if os.environ.get('FLASK_ENV') == 'development' else False
```

Comment:
Just a quickie suggestion, you might also do this - `DEBUG = os.environ.get('FLASK_ENV') == 'development'`

Reply:
thanks for you advice